src/pages/dashboardUserEdit/EditUserRegistryPage.tsx

```diff
@@ -5,10 +5,11 @@ import { resolvePathVariables } from '@pagopa/selfcare-common-frontend/utils/rou
 import { useTranslation } from 'react-i18next';
 import ProductNavigationBar from '../../components/ProductNavigationBar';
 import { DASHBOARD_USERS_ROUTES } from '../../routes';
-import withUserDetail, { withUserDetailProps } from '../../decorators/withUserDetail';
+// import withUserDetail, { withUserDetailProps } from '../../decorators/withUserDetail';
+import withUserRegistry, { withUserRegistryProps } from '../../decorators/withUserRegistry';
```

Comment:
let the code clean if not usefull
```suggestion
import withUserRegistry, { withUserRegistryProps } from '../../decorators/withUserRegistry';
```